### PR TITLE
fix: Explore on other networks profile button

### LINKED ISSUE #9069: Explore on other networks profile button
image

- lets make the button primary, it will have more contrast and maybe people will use it :D 
primary rounded please 
image

https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=5311-18771&mode=design&t=vaN5lU7ICdsxH9ph-11

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #9069
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI
image
image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by exezbcz** [comment]
uwu nice, even comes with a component 


thanks

File: libs/ui/src/components/NeoButton/NeoButton.scss [hunk 1]
```diff
@@ -106,6 +106,21 @@
         }
       }
     }
+    &--primary-rounded {
```
Comment (roiLeo) [inline libs/ui/src/components/NeoButton/NeoButton.scss:109]: why create a variant when you just had to call `primary` & `rounded` props, would be top to be based on #8362

**Review by roiLeo — COMMENTED** [review]
I'd like to see an alternative inside `libs/ui`

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 6.67 USD/DOT ~ 2.999 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0xe22a7e4216e815656dd24e1a0ad71793295dd565b81ca9b7bb309f54a4eebbe7](https://polkadot.subscan.io/extrinsic/0xe22a7e4216e815656dd24e1a0ad71793295dd565b81ca9b7bb309f54a4eebbe7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**